src/advise.js

```diff
@@ -107,9 +107,9 @@ Coverage of language translations might be not accurate for the repository in qu
 	reOrderFail: `It appears something went wrong with the reordering of the bump commit.
 
 You really shouldn't be seeing this message, so, if you are, ping someone in engineering to see if they can help you figure out what went wrong.`,
-	defaultBranch: `Sorry, we were unable to read from your 'upstream' remote.
+	defaultBranch: `Sorry, we were unable to determine if 'master' or 'main' is your default branch.
```

Comment:
Hmm, I wonder if we should reword this. Technically it's not the default branch... case in point, `web-lightning-ui` default branch is `develop`

I think maybe something like "root branch" or "base branch" might be more accurate?